README.md

```
@@ -2,236 +2,137 @@
 
-```bash
-# Acknowledge a violation
-@watchflow acknowledge "Documentation updates only, no code changes"
-@watchflow ack "Documentation updates only, no code changes"
+## Comment commands
 
-# Acknowledge with reasoning
-@watchflow acknowledge "Emergency fix, team is unavailable"
-@watchflow ack "Emergency fix, team is unavailable"
+| Command | Purpose |
+|--------|--------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)